.circleci/config.yml

```diff
@@ -134,11 +134,12 @@ jobs:
       - store_artifacts: {path: ./build/devilutionx, destination: devilutionx_m68k}
   vita:
     docker:
-      - image: sarcasticat/dolcesdk:latest
+      - image: vitasdk/vitasdk:latest
     working_directory: ~/repo
     steps:
       - checkout
-      - run: cmake -S. -Bbuild -GNinja .. -DBINARY_RELEASE=ON -DVITA=1
+      - run: apk --no-cache add cmake ninja
+      - run: cmake -S. -Bbuild -GNinja -DCMAKE_TOOLCHAIN_FILE=${VITASDK}/share/vita.toolchain.cmake -DBINARY_RELEASE=ON
```

Comment:
```suggestion
      - run: cmake -S. -Bbuild -GNinja -DCMAKE_TOOLCHAIN_FILE=${VITASDK}/share/vita.toolchain.cmake  -DCMAKE_BUILD_TYPE=Release
```
or
```suggestion
      - run: cmake -S. -Bbuild -GNinja -DCMAKE_TOOLCHAIN_FILE=${VITASDK}/share/vita.toolchain.cmake -DNIGHTLY_BUILD=ON
```

Reply:
Did second one, to be consistent with other platforms (except switch, which i used or example)

Reply:
Yeah, the switch was affected by an issue where it couldn't compile under normal conditions, I believe the issue has since been resolved, but on top of that, the latest container broke compiling completely. Don't know if they resolved it since then, but generally, the Switch port still could do a lot of cleaning. Some improvements are available upstream, some should be made upstream and we should try and align a bit more with how DevKitPro prefers that things be setup.